<launch>
  <node pkg="ultraspeech" name="logger" type="logger.py"/>

  <node pkg="ultraspeech" name="subjectUI" type="subjectUI.py" args="/home/robotlab/ros/ua-ros-pkg/ua_experimental/ultraspeech/stimuli.txt 5"/>

  <node pkg="ultraspeech" name="mic1_capture" type="jack_audio_capture" args="mic1_capture 1">
    <remap from="audio" to="mic1_capture/audio"/>
  </node>
  <node pkg="ultraspeech" name="mic1_save" type="audio_clip_writer" args="mic1">
    <remap from="audio" to="mic1_capture/audio"/>
    <remap from="save_name" to="mic1_capture/save_name"/>
  </node>  

</launch>